### PR TITLE
Mergeup 5.5.x to master

### PR DESCRIPTION
**Pull request by jhelwig**

### CODE REVIEW

File: acceptance/README.md [hunk 1]
```diff
@@ -47,11 +47,12 @@ Beaker will install the following:
 
 This corresponds to installing facter and hiera stable from our internal mirror, while installing puppet SHA abcd from a git daemon on my local machine percival.  See below for details on setting up a local git daemon.
 
-Running Tests on the vcloud
+Running Tests on the vcloud (now vmpooler)
 ---------------------------
 
-In order to use the Puppet Labs vcloud, you'll need to be a Puppet Labs employee.
-Community members should see the [guide to running the tests on vagrant boxen](#running-tests-on-vagrant-boxen).
+Only applies to Pulletlabs employees. Community members should see the [guide to running the tests on vagrant boxen](#running-tests-on-vagrant-boxen).
```
Comment (shrug) [inline acceptance/README.md]: Typo "Pulletlabs"

Comment (shrug) [reply acceptance/README.md]: I see that came in from the 5.5.x branch....

Comment (jhelwig) [reply acceptance/README.md]: I'm adding a maint commit on top to fix the typo.

Comment (jhelwig) [reply acceptance/README.md]: Actually, it can be its own PR to get merged up.

**Pull request merged**